README.md

```diff
@@ -1,18 +1,26 @@
 # Building With Adobe DRM
 
+## Building the Application
+
 01. `git clone https://github.com/NYPL/Simplified-iOS.git` or `git clone git@github.com:NYPL-Simplified/Simplified-iOS.git`
 02. `cd Simplified-iOS`
 03. `git submodule update --init --recursive`
 04. Install [Cocoapods](https://cocoapods.org) if you haven't already.
 05. `pod install`
 06. Symlink the "DRM_Connector_Prerelease" directory to "adobe-rmsdk" within the "Simplified-iOS" directory. (You will need to have obtained the Adobe DRM Connector prerelease from Adobe.)
-07. Follow the instructions in "adobe-rmsdk/RMSDK_User_Manual(obj).pdf" to build OpenSSL (section 12.1) and cURL (section 12.3).*
+07. Build OpenSSL and cURL as described in the following "Building OpenSSL and cURL" section. Ensure you're in the "Simplified-iOS" directory before continuing to the next step.
 08. `sh adobe-rmsdk-build.sh`
 09. `cp APIKeys.swift.example Simplified/APIKeys.swift` and edit accordingly.
 10. `open Simplified.xcworkspace`
 11. Build.
 
-\* If you want to release your app in Apple's App Store, you must add `--enable-ipv6` to line 80 of Adobe's "build.sh" script used for building cURL. Without that flag, cURL will not provide proper support for IPv6 and therefore Apple will reject your application.
+## Building OpenSSL and cURL
+
+Follow the instructions in "adobe-rmsdk/RMSDK_User_Manual(obj).pdf" to build OpenSSL (section 12.1) and cURL (section 12.3). Be sure to note the following:
+
+* You will need to verify and edit the "build.sh" scripts for both OpenSSL and cURL to reflect the correct version numbers and local directory names (lines 11–24).
+* You must add `--enable-ipv6` to line 80 of Adobe's "build.sh" script used for building cURL. This necessary both due to Apple's requirements for IPv6 support and because the library may not build against recent iOS SDKs otherwise.
```

Comment:
I have notes saved: "values to edit in build.sh before building:
OPENSSL_VERSION=“1.0.2m”
SDK_VERSION
min_version

tell them to make the public/ios directory to copy the include and lib into"